plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtils.java

```diff
@@ -130,6 +132,27 @@ public static Dictionary convertAssetPackLocationsToDictionary(
             (d1, d2) -> d1.putAll(d2));
   }
 
+  /**
+   * Serializes an Exception object into Godot Dictionary. If the Exception is not an
+   * AssetPackException, the errorCode entry in returnDict will be set to INTERNAL_ERROR.
+   *
+   * @param e Exception to be converted to Dictionary
+   * @return serialized Dictionary
+   */
+  public static Dictionary convertExceptionToDictionary(Exception e) {
+    Dictionary returnDict = new Dictionary();
+    returnDict.put("type", e.getClass().getCanonicalName());
```

Comment:
It'd be good to make static final variables for the exception dictionary fields.

Reply:
Convert these magic ~~numbers~~ Strings to static final Strings